cockatrice/src/chatview/chatview.cpp

```diff
@@ -145,76 +152,86 @@ void ChatView::appendUrlTag(QTextCursor &cursor, QString url)
 
 void ChatView::appendMessage(QString message,
                              RoomMessageTypeFlags messageType,
-                             QString sender,
+                             const QString &userName,
                              UserLevelFlags userLevel,
                              QString UserPrivLevel,
                              bool playerBold)
 {
     bool atBottom = verticalScrollBar()->value() >= verticalScrollBar()->maximum();
-    bool sameSender = (sender == lastSender) && !lastSender.isEmpty();
+    // messageType should be Event_RoomSay::UserMessage though we don't actually check
+    bool isUserMessage = !(userName.toLower() == "servatrice" || userName.isEmpty());
```

Comment:
double hardcoded name, make into a const

Reply:
I don't like this name to begin with, I wanted to leave this check as similar as possible, when is "servatrice" talking anyway? only when the user talks or will it actually set it to servatrice otherwise?